Describe Warning 1016 (#1985)
DevExpress/devextreme-documentation: api-reference/10 UI Components/Errors and Warnings/W1016.md

---
---
##### shortDescription

> Occurs if you specify the deprecated **formatName** and **formatValues** fields in the [items](/Documentation/ApiReference/UI_Components/dxHtmlEditor/Configuration/toolbar/items/) array passed to the  [HtmlEditor](/Documentation/ApiReference/UI_Components/dxHtmlEditor)'s [toolbar](/Documentation/ApiReference/UI_Components/dxHtmlEditor/Configuration/toolbar/) property.

Reviewer: ```suggestion
Occurs if you specify the deprecated **formatName** and **formatValues** properties in the [items](/Documentation/ApiReference/UI_Components/dxHtmlEditor/Configuration/toolbar/items/) array within the [HtmlEditor](/Documentation/ApiReference/UI_Components/dxHtmlEditor)'s [toolbar](/Documentation/ApiReference/UI_Components/dxHtmlEditor/Configuration/toolbar/) property.
```

---

> Use the **name** and **acceptedValues** fields as demonstrated below:

Reviewer: ```suggestion
Use the **name** and **acceptedValues** properties instead:
```
